modules/apps/dynamic-data-mapping/dynamic-data-mapping-api/src/main/java/com/liferay/dynamic/data/mapping/form/validation/util/DateParameterUtil.java

```diff
@@ -62,8 +68,25 @@ public static LocalDateTime getLocalDateTime(String dateTimeString) {
 			return null;
 		}
 
-		return LocalDateTime.parse(
-			dateTimeString, DateTimeFormatter.ofPattern("yyyy-MM-dd H:mm"));
+		String parseException = null;
+
+		for (String datePattern : datePatterns) {
+			try {
+				return LocalDateTime.parse(
+					dateTimeString, DateTimeFormatter.ofPattern(datePattern));
+			}
+			catch (DateTimeParseException dateTimeParseException) {
+				parseException = String.valueOf(dateTimeParseException);
+			}
+		}
+
+		if (parseException != null) {
+			_log.error(parseException);
+		}
+
+		return getLocalDate(
```

Comment:
In this case, does it make sense to the end user if we are defaulting to comparing with the start of the day? I think it can be confusing, and maybe its better to return null or an error, like the `getLocalDate` method